src/test/java/org/openmrs/contrib/qaframework/RunTest.java

```diff
@@ -41,14 +42,25 @@
 		public static final String VITALS = "@vitals";
 		public static final String FIND_PATIENT = "@findPatient";
 		public static final String SELENIUM_LOGIN = SELENIUM + " and " + LOGIN;
-		public static final String SELENIUM_DASHBOARD = SELENIUM + " and " + DASHBOARD;
-		public static final String SELENIUM_INITIAL_SETUP = SELENIUM + " and " + INITIAL_SETUP;
-		public static final String SELENIUM_VITALS = SELENIUM + " and " + VITALS;
-		public static final String SELENIUM_CLINICAL_VISIT = SELENIUM + " and " + CLINICAL_VISIT;
-		public static final String SELENIUM_ENCOUNTER = SELENIUM + " and " + INPATIENT;
-		public static final String SELENIUM_REGISTRATION = SELENIUM + " and " + REGISTRATION;
-		public static final String SELENIUM_FIND_PATIENT = SELENIUM + " and " + FIND_PATIENT;
-		public static final String SELENIUM_USER_ACCOUNT = SELENIUM + " and " + USER_ACCOUNT;
-		public static final String SELENIUM_REPORT = SELENIUM + " and " + REPORT;
+		public static final String SELENIUM_CONDITION = SELENIUM + " and "
+				+ CONDITION;
+		public static final String SELENIUM_DASHBOARD = SELENIUM + " and "
+				+ DASHBOARD;
+		public static final String SELENIUM_INITIAL_SETUP = SELENIUM + " and "
+				+ INITIAL_SETUP;
+		public static final String SELENIUM_VITALS = SELENIUM + " and "
+				+ VITALS;
+		public static final String SELENIUM_CLINICAL_VISIT = SELENIUM + " and "
+				+ CLINICAL_VISIT;
+		public static final String SELENIUM_ENCOUNTER = SELENIUM + " and "
+				+ INPATIENT;
+		public static final String SELENIUM_REGISTRATION = SELENIUM + " and "
+				+ REGISTRATION;
+		public static final String SELENIUM_FIND_PATIENT = SELENIUM + " and "
+				+ FIND_PATIENT;
+		public static final String SELENIUM_USER_ACCOUNT = SELENIUM + " and "
+				+ USER_ACCOUNT;
+		public static final String SELENIUM_REPORT = SELENIUM + " and "
+				+ REPORT;
```

Comment:
you need to update your branch with changes from master!

Reply:
okay thanks